xbmc/cores/AudioEngine/Utils/AEBuffer.cpp

```diff
@@ -20,9 +20,9 @@
 
 #include "AEBuffer.h"
 #include <algorithm>
-#ifdef TARGET_POSIX
```

Comment:
why don't you just remove entire CAEBuffer? it is not used anymore (at least I could not find any usage)